src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -115,7 +115,7 @@ public static Calendar now(double timeZone) {
      * Convert a date string and time string into a Date object. Returns null on error.
      *
      * @param inputDate
-     *            The date in format dd/MM/yyyy
+     *            The date in format D, dd MMM, yyyy
```

Comment:
There's a discrepancy between the format here and the one actually used in the helper method below?